en/docs/design-your-project/index.md

```diff
@@ -1,4 +1,4 @@
-# Architecture Diagram
+# Design your project
```

Comment:
Seems `Learn` part is unnecessary in `Learn related concepts` sub title.

Reply:
Removed.